src/test/scala/org/zalando/zhewbacca/RequestValidatorSpec.scala

```diff
@@ -9,18 +9,18 @@ import scala.concurrent.{Await, Future}
 import scala.concurrent.ExecutionContext.Implicits.global
 class RequestValidatorSpec extends Specification {
 
-  val TestTokenInfo = TokenInfo("", Scope.Empty, "token type", "user uid")
+  val testTokenInfo = TokenInfo("", Scope.Empty, "token type", "user uid", realm = "/employees")
 
   "Request Validator" should {
     "provide token information when token is valid" in {
       val authProvider = new AuthProvider {
         override def valid(token: Option[OAuth2Token], scope: Scope): Future[AuthResult] =
-          Future.successful(AuthTokenValid(TestTokenInfo))
+          Future.successful(AuthTokenValid(testTokenInfo))
       }
 
       val result = Await.result(RequestValidator.validate(Scope(Set("uid")), FakeRequest(), authProvider), 1.seconds)
       result.isRight must beTrue
-      result.right.get must beEqualTo(TestTokenInfo)
+      result.right.get must beEqualTo(testTokenInfo)
```

Comment:
Codacy is complaining "Usage of get on optional type."
Can't you try with
`result.right must beEqualTo(Some(testTokenInfo))`

Reply:
That leads to failing tests, because `right` here is not an Option, but a `RightProjection`.

I'll use `result must beEqualTo(Right(testTokenInfo))` instead. (I could also use `result.right.toOption must beEqualTo(Some(testTokenInfo))`, but that looks worse.)